packages/angular/src/tracing.ts

```diff
@@ -143,10 +143,13 @@ export class TraceService implements OnDestroy {
         (event.state as unknown as RouterState & { root: ActivatedRouteSnapshot }).root,
       );
 
+      if (route) {
+        getCurrentScope().setTransactionName(route);
+      }
+
       const activeSpan = getActiveSpan();
       const rootSpan = activeSpan && getRootSpan(activeSpan);
 
-      // TODO (v8 / #5416): revisit the source condition. Do we want to make the parameterized route the default?
```

Comment:
I opted to remove this comment because it's very unlikely that we change the behaviour (now extracted to `_updateSpanAttributesForParametrizedUrl`) and even if, it'd be a global initiative across all span update occasions.